rfcs/packaging/communicating-convergence-status.md

```diff
@@ -0,0 +1,141 @@
+| `@fluentui/react-cards` | Weird case since there's an old version (built on `@uifabric/foundation` `createComponent` patterns) but it never officially graduated from experimental status and isn't exported from the suite. It also has a `next` folder where the converged version is currently being worked on. |
+| `@fluentui/react-date-time` | Previously `@uifabric/date-time`. It's a rewrite (mostly by OWA folks, started awhile ago) of the original OUFR Calendar and DatePicker, but still using older patterns. New for v8, the Calendar and DatePicker from this package are exported from `@fluentui/react` by default. |
+| `@fluentui/react-focus` | Current version of FocusZone with most of the work needed to converge with v0's FocusZone. Still differs in are some default values and things that are pulled from the theme. |
+| `@fluentui/react-checkbox`<br/>`@fluentui/react-link`<br/>`@fluentui/react-slider`<br/>`@fluentui/react-toggle`<br/>`@fluentui/react-tabs` | These previously default-exported the non-converged (v8) versions of the respective components (with prototype converged versions under `src/next`). The v8 components were recently moved backed to the suite and the convergence work was moved to the root. |
+| `@fluentui/react-avatar`<br/>`@fluentui/react-flex`<br/>`@fluentui/react-image`<br/>`@fluentui/react-text` | These only contain work on the converged component versions (in varying states of active work and completeness). |
+| `@fluentui/react-charting` | Has active work ongoing but is owned by a different team and doesn't really follow any of the newer patterns we're working on. (Once we establish a pattern for "high-value component" repos, we may split this out.) |
+| `@fluentui/react-experiments` | Weird legacy package which we don't have a good story for moving forward, and probably don't recommend (certainly don't promote) for use except by authoring teams of the individual components. It's a mix of components in active development or heavy use by partners, old trials of previous patterns we should definitely delete, and things of unknown status or abandoned. [More discussion here.](https://github.com/microsoft/fluentui/issues/16452) |
+<!-- prettier-ignore-end -->
+
+## Detailed Design or Proposal
+
+### All v8 components live in `@fluentui/react`
+
+Motivation: improve contribution story, in some ways improve consumption story.
+
+- Get rid of `react-internal` _(see detailed discussion below)_
+  - Move all components back into `react`
+  - To break a cycle: move `react-date-time` components into `react`, and have `react-date-time` re-export them
+- _(done)_ ~~Move v8 Checkbox, Link, Pivot, Slider, Toggle out of component packages and back to `react`~~
```

Comment:
maybe out of topic but I'm bit confused here.  Please help me understand what's the workflow. This is an RFC. It's not merged yet and we already do things in master thus making this an Issue/Epic rather than RFC (is this something that is common or that we plan to do going forward ?). thx!

Reply:
This was an RFC, but due to time constraints trying to (finally) get v8 out the door, and the unexpectedly extensive discussion that arose around the RFC, I already started implementing some of the parts that we were more certain about.

Reply:
We're still figuring out what our process will be around RFCs overall, so anything that I did here shouldn't necessarily be viewed as prescriptive.

Reply:
@Hotell has a good point (and @ecraig12345 is also right that we're figuring it out 😄) 

Going forward I would hope that this won't be much of a problem. Ideally there should have been an RFC a few months ago calling out all this, so we wouldn't be discussing it while we were implementing it. 

I'd recommend we don't make any adjustments right now (see [Don’t scar on the first cut](https://m.signalvnoise.com/dont-scar-on-the-first-cut)). 

If this continues to cause problems we could say that any pending work should either be called out clearly or in a linked epic issue.

Reply:
sure thingy and makes sense, thanks for clarification 🍺